physical_properties/pKa/experimental_data/pKa_replicate_experimental_results.csv

```diff
@@ -0,0 +1 @@
SM15,4.69,8.92,,UV-metric pKa,17K-10011,M16,17K-10011_M16_UV-metric pKa_report
SM16,5.37,10.64,,UV-metric pKa,17J-03025,M18,17J-03025_M18_UV-metric pKa_report
SM16,5.37,10.65,,UV-metric pKa,17J-03026,M18,17J-03026_M18_UV-metric pKa_report
SM16,5.38,10.65,,UV-metric pKa,17J-03027,M18,17J-03027_M18_UV-metric pKa_report
SM17,3.15,,,UV-metric pKa,17J-02024,M19,17J-02024_M19_UV-metric pKa_report
SM17,3.16,,,UV-metric pKa,17J-02025,M19,17J-02025_M19_UV-metric pKa_report
SM17,3.17,,,UV-metric pKa,17J-02026,M19,17J-02026_M19_UV-metric pKa_report
SM18,2.10,9.51,10.93,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28001,D01,17I-28001_D01_UV-metric psKa_report
SM18,2.15,9.59,11.03,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28002,D01,17I-28002_D01_UV-metric psKa_report
SM18,2.19,9.63,11.11,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28003,D01,17I-28003_D01_UV-metric psKa_report
SM19,9.59,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28004,D02,17I-28004_D02_UV-metric psKa_report
SM19,9.58,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28005,D02,17I-28005_D02_UV-metric psKa_report
SM19,9.52,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17I-28006,D02,17I-28006_D02_UV-metric psKa_report
SM20,5.72,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-12002,D05,17J-12002_D05_UV-metric psKa_report
SM20,5.75,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-12003,D05,17J-12003_D05_UV-metric psKa_report
SM20,5.64,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17K-16013,D05,17K-16013_D05_UV-metric psKa_report
SM21,4.11,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06002,D06,17J-06002_D06_UV-metric psKa_report
SM21,4.09,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06003,D06,17J-06003_D06_UV-metric psKa_report
SM21,4.09,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06004,D06,17J-06004_D06_UV-metric psKa_report
SM22,2.45,7.42,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07003,D07,17J-07003_D07_UV-metric psKa_report
SM22,2.37,7.45,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07004,D07,17J-07004_D07_UV-metric psKa_report
SM22,2.37,7.41,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07005,D07,17J-07005_D07_UV-metric psKa_report
SM23,5.45,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07006,D08,17J-07006_D08_UV-metric psKa_report
SM23,5.44,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07007,D08,17J-07007_D08_UV-metric psKa_report
SM23,5.45,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-07008,D08,17J-07008_D08_UV-metric psKa_report
SM24,2.61,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06007,D09,17J-06007_D09_UV-metric psKa_report
SM24,2.58,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06008,D09,17J-06008_D09_UV-metric psKa_report
SM24,2.60,,,"UV-metric psKa, Yasuda-Shedlovsky Extrapolation",17J-06009,D09,17J-06009_D09_UV-metric psKa_report
```

Comment:
Same commend as on `experimental_ID_and_SAMPL6_molecule_ID_table.csv` why is this a single line? Could you explain in the file how we're supposed to read this?

Reply:
This file was supposed to be a CSV with two columns. I corrected it.